index.js

```diff
@@ -15,6 +15,7 @@ program
   .description(packageJSON.description)
 
 program
+  .addOption(new Option('-u, --token-usage', 'specify if you want to see tokens that were sent in the prompt and the number of tokens that were returned'))

```

Comment:
I tired to keep all the flags alphabetical in this file so I would prefer it that way.

Reply:
Fixed with https://github.com/aamfahim/explainer.js/pull/23/commits/7b0c2f2c471cb50bd15c4f8c1a2011b8bcf9e2e4